### DISCUSSION TITLE
Docker container for Observer / player

### DISCUSSION THREAD

**Discussion by btelliot** [openbroadcaster/observer#63] [General]
Hi folks,

I'm trying to get a development environment set up for Observer.  Does anyone have a docker image they can share?

Thanks,

**Comment by brookgagnon** [comment]
I think someone from our team is working on this but don't have the details.

If you can have a fresh Ubuntu 20.04 install with an FQDN (subdomain / domain) to use, you can try this installer:
https://github.com/brookgagnon/ob-install

Note it's alpha, might break or catch fire, and doesn't validate user inputs (so be careful to enter your settings carefully). I set this up as a quick temporary measure for fresh OB installs for development/testing purposes.

If you do try it out, let me know how it goes. :)

**Comment by btelliot** [comment]
Awesome thanks Brook. I’ll probably reach out sometime to pick your brain
about the software, but I need to get a working version up and running
first!

**Comment by btelliot** [comment]
Hi Brook,

I finally got around to trying this today with docker.

Here are the commands I ran:
from Command Line
docker pull ubuntu:focal

from CLI
apt update && apt upgrade
apt install wget
wget
https://raw.githubusercontent.com/brookgagnon/ob-install/bb9852691d4a7303b027c8655bae87537e028ca1/ubuntu-focal.sh
su
sh ubuntu-focal.sh

I'm running into the same problem I have when I try to do other installs,
and that is that forwhatever reason, my ubuntu focal cannot add the
repositories for PHP and nginx.  See below:
add-apt-repository ppa:ondrej/php -y
add-apt-repository ppa:ondrej/nginx -y

[image: image.png]
[image: image.png]
